src/ansys/fluent/core/utils/file_transfer_service.py

```diff
@@ -14,9 +14,7 @@ def __init__(self):
 class PimFileTransferService:
     """Instantiates a file uploader and downloader to have a seamless file reading /
     writing in the cloud particularly in Ansys lab . Here we are exposing upload and
```

Comment:
- Mention of session objects not required in this module. This module is independent of the session classes.
- Ansys Lab is one specific PyPIM user, so we can omit mentioning it.
- Don't need to specify that it's _internal_. The intention is that it is _externally_ usable. The infrastructure here will be / is being installed at customer private clouds etc.
- Comment can be much shorter - just ensure no space before the full stop.

Reply:
Removed. Thank you.